influxdb2_operational_monitoring/influxdb2_cardinality_now.yml

```diff
@@ -40,13 +64,30 @@ spec:
     type: query
 ---
 apiVersion: influxdata.com/v2alpha1
+kind: Variable
```

Comment:
what is this for?  Seems like maybe it snuck in unintentionally

Reply:
Was originally there, but moved to different area of file